assets/texts/fr/amoDescription.html

```diff
@@ -0,0 +1,24 @@
+Ce module complémentaire (extension) pour Firefox vous donne un bon petit sélecteur d'émoticônes pour en trouver et en sélectionner. Il est copié dans votre presse-papier quand vous cliquez dessus.
```

Comment:
Also the whole `amoDescription.html` file is outdated, e.g. (It was mostly rewritten.)

Reply:
By the way, shouldn't the "good for first issue" be used to award newcomers for doing a good new issue? 🤨